src/OpenIddict.EF/OpenIddictContext.cs

```diff
@@ -10,29 +10,120 @@
 using OpenIddict.Models;
 
 namespace OpenIddict {
-    public class OpenIddictContext<TUser, TApplication, TRole, TKey> : IdentityDbContext<TUser, TRole, TKey>
+    /// <summary>
+    /// Base class for the Entity Framework database context used for OpenIddict.
+    /// </summary>
+    /// <typeparam name="TApplication">The type of application objects.</typeparam>
+    /// <typeparam name="TApplicationKey">The type used for the primary key for the application.</typeparam>
+    /// <typeparam name="TUser">The type of user objects.</typeparam>
+    /// <typeparam name="TRole">The type of role objects.</typeparam>
+    /// <typeparam name="TKey">The type of the primary key for users and roles.</typeparam>
+    /// <seealso cref="Microsoft.AspNetCore.Identity.EntityFrameworkCore.IdentityDbContext{TUser, TRole, TKey}" />
+    public class OpenIddictContext<TApplication, TApplicationKey, TUser, TRole, TKey> : IdentityDbContext<TUser, TRole, TKey>
```

Comment:
These entities are coming as part of my PR. E.g there's a direct relationship between `OpenIddictAuthorization` and `OpenIddictToken`:

```csharp
/// <summary>
/// Represents an OpenIddict authorization.
/// </summary>
public class OpenIddictAuthorization<TToken, TKey> where TKey : IEquatable<TKey> {
    /// <summary>
    /// Gets or sets the unique identifier
    /// associated with the current authorization.
    /// </summary>
    public virtual TKey Id { get; set; }

    /// <summary>
    /// Gets or sets the space-delimited scopes
    /// associated with the current authorization.
    /// </summary>
    public virtual string Scope { get; set; }

    /// <summary>
    /// Gets or sets the list of tokens
    /// associated with the current authorization.
    /// </summary>
    public virtual IList<TToken> Tokens { get; } = new List<TToken>();

    /// <summary>
    /// Gets or sets the identifier of the user profile
    /// associated with the current authorization.
    /// </summary>
    public virtual TKey UserId { get; set; }
}
```

```csharp
/// <summary>
/// Represents an OpenIddict token.
/// </summary>
public class OpenIddictToken<TKey> where TKey : IEquatable<TKey> {
    /// <summary>
    /// Gets or sets the identifier of the authorization attached with the current token.
    /// This property may be null if the token was issued without
    /// requiring the user consent or is bound to a client application.
    /// </summary>
    public virtual TKey AuthorizationId { get; set; }

    /// <summary>
    /// Gets or sets the unique identifier
    /// associated with the current token.
    /// </summary>
    public virtual TKey Id { get; set; }

    /// <summary>
    /// Gets or sets the type of the current token.
    /// </summary>
    public virtual string Type { get; set; }

    /// <summary>
    /// Gets or sets the identifier of the user attached with the current token.
    /// This property is null if the token represents a client application.
    /// </summary>
    public virtual TKey UserId { get; set; }
}
```